ts/components/session/settings/SessionSettings.tsx

```diff
@@ -406,7 +410,28 @@ class SettingsViewInner extends React.Component<SettingsViewProps, State> {
         comparisonValue: undefined,
         onClick: undefined,
       },
+      {
+        id: 'message-request-setting',
+        title: 'Message Requests', // TODO: translation
+        description: 'Enable Message Request Inbox',
+        hidden: false,
+        type: SessionSettingType.Toggle,
+        category: SessionSettingCategory.Appearance,
+        setFn: () => {
+          window.inboxStore?.dispatch(toggleAudioAutoplay());
 
+          if (window.inboxStore?.getState().userConfig.messageRequests) {
+            window.inboxStore?.dispatch(disableMessageRequests());
+          } else {
+            window.inboxStore?.dispatch(enableMessageRequests());
```

Comment:
calling the `window.inboxStore?.getState()` is pretty dirty. Why not have an action `toggleShowMessageRequests` which will have knowlefge of the current state?